doc_source/security-groups-for-pods.md

```diff
@@ -6,7 +6,7 @@ Security groups for Pods integrate Amazon EC2 security groups with Kubernetes Po
 + Before deploying security groups for Pods, consider the following limitations and conditions:
 + Security groups for Pods can't be used with Windows nodes\.
 + Security groups for Pods can be used with clusters configured for the `IPv6` family that contain Amazon EC2 nodes by using version 1\.16\.0 or later of the Amazon VPC CNI plugin\. You can use security groups for Pods with clusters configure `IPv6` family that contain only Fargate nodes by using version 1\.7\.7 or later of the Amazon VPC CNI plugin\. For more information, see [`IPv6` addresses for clusters, Pods, and services](cni-ipv6.md)
-+ Security groups for Pods are supported by most [Nitro\-based](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-types.html#ec2-nitro-instances) Amazon EC2 instance families, though not by all generations of a family\. For example, the `m5`, `c5`, `r5`, `p3`, `m6g`, `c6g`, and `r6g` instance family and generations are supported\. No instance types in the `t` family are supported\. For a complete list of supported instance types, see the [limits\.go](https://github.com/aws/amazon-vpc-resource-controller-k8s/blob/release-1.1.4/pkg/aws/vpc/limits.go) file on Github\. Your nodes must be one of the listed instance types that have `IsTrunkingCompatible: true` in that file\.
++ Security groups for Pods are supported by most [Nitro\-based](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-types.html#ec2-nitro-instances) Amazon EC2 instance families, though not by all generations of a family\. For example, the `m5`, `c5`, `r5`, `m6g`, `c6g`, and `r6g` instance family and generations are supported\. No instance types in the `t` family are supported\. For a complete list of supported instance types, see the `limits.go` file linked in `Notice` for the relevant [release](https://github.com/aws/amazon-vpc-resource-controller-k8s/releases) on Github\. Your nodes must be one of the listed instance types that have `IsTrunkingCompatible: true` in that file\.
```

Comment:
Let's keep the link directly to the limits.go and just increment the version to 1.4.5/1.4.6 (1.4.6 is tagged in that repo but not released).
as in: `https://github.com/aws/amazon-vpc-resource-controller-k8s/blob/v1.4.6/pkg/aws/vpc/limits.go`

Linking to other text that has links in it doesn't work out; many folks don't want to follow multiple steps to leave the docs before getting to the info. I can afford the extra step every few weeks to increment this version when you release.

At a later date, I'll revisit doing some "transclude" to copy data in from other GitHub sources - incrementing these versions, converting JSON to lists like this limits.go, pulling info from APIs, etc.